Packs/TAXIIServer/Integrations/TAXIIServer/TAXIIServer.yml

```diff
@@ -0,0 +1,118 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: TAXIIServer
+  version: -1
+configuration:
+- additionalinfo: The query to run to update its EDL. To view expected results, you
+    can run the following command from the Demisto CLI `!findIndicators query=<your
+    query>`
+  display: Indicator Query
+  name: indicators_query
+  required: false
+  type: 0
+- additionalinfo: Maximum number of items in the EDL
+  defaultvalue: '2500'
+  display: EDL Size
+  hidden: false
+  name: edl_size
+  required: true
+  type: 0
+- additionalinfo: Enabling this will prevent automatic EDL refresh
+  display: Update EDL On Demand Only
+  name: on_demand
+  required: false
+  type: 8
+- additionalinfo: How often to refresh the EDL
+  defaultvalue: 5 minutes
+  display: Refresh Rate
+  name: cache_refresh_rate
+  required: false
+  type: 0
+- defaultvalue: 'true'
+  display: Long Running Instance
+  name: longRunning
+  required: false
+  type: 8
+- additionalinfo: Will run the EDL service on this port from within Demisto
+  defaultvalue: '7000'
+  display: Listen Port
+  name: longRunningPort
+  required: true
+  type: 0
+- display: Certificate (Required for HTTPS)
+  name: certificate
+  required: false
+  type: 12
+- display: Private Key (Required for HTTPS)
+  name: key
+  required: false
+  type: 14
+- additionalinfo: JSON string of users in a username - password
+  display: Users JSON
+  name: users
+  required: false
+  type: 14
+- additionalinfo: Ignores certificate and private key, and will run the EDL service
```

Comment:
are you sure its meant for that? i think this flag is meant to spin up both http and https server